.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,48 @@
+name: pn-codegen docker image build and publish
+
+on:
+  push:
+    branches: [ "develop", "main" ]
+  pull_request:
```

Comment:
Attenzione le pull request possono essere fatte anche da utenti non di questo repository.

Dai un'occhiata a 
https://pagopa.atlassian.net/wiki/spaces/PN/pages/471860352/Sicurezza+delle+pipeline punto 1

Reply:
Secondo l'impostazione di default l'action viene triggerata solo su approvazione in caso di PR esterna:
<img width="902" alt="Schermata 2023-03-10 alle 10 31 20" src="https://user-images.githubusercontent.com/958743/224279657-5caade1d-1343-422f-86b2-a337458500e1.png">

Reply:
ok, grazie.
Imparo cose nuove